File: idakonyvtar/src/main/java/eu/eyan/idakonyvtar/util/OkCancelDialog.java
package eu.eyan.idakonyvtar.util;

import java.awt.Window;

import javax.swing.JDialog;

public class OkCancelDialog extends JDialog {
	public OkCancelDialog(Window owner) {
		super(owner);
	}

	public boolean isOk() {
		return ok;
	}

	public void setOk(boolean ok) {
		this.ok = ok;
	}

	private static final long serialVersionUID = 1L;

	private boolean ok = false;
}

File: idakonyvtar/src/main/scala/eu/eyan/idakonyvtar/util/OkCancelDialog.scala
package eu.eyan.idakonyvtar.util;

import java.awt.Window;

import javax.swing.JDialog;
import java.awt.Dialog.ModalityType

class OkCancelDialog(owner: Window) extends JDialog(owner) {
  var ok = false;
  def isOk() = ok
  def setOk(ok: Boolean) = this.ok = ok
}